service/test/agama/dbus/storage/interfaces/md_examples.rb

```diff
@@ -45,9 +45,13 @@
       end
     end
 
-    describe "#md_members" do
-      it "returns the name of the MD members" do
-        expect(subject.md_members).to contain_exactly("/dev/sda1", "/dev/sda2")
+    describe "#md_devices" do
+      it "returns the D-Bus path of the MD components" do
```

Comment:
BTW all those changes to dbus API makes me worry that I do not see update of documentation, but looks like device interfaces are not documented at all.

Reply:
Exactly, we don't have documentation yet for that interfaces. In fact, I don't know how to fit them in our system for documenting D-Bus, because those interfaces are not associated to a specific object but to an arbitrary object in the tree of devices. Maybe @mvidner could help.